File: 2020/Java/Junior Problems/J1.java
import java.util.Scanner;
public class Main
{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		 int S =sc.nextInt();
		 int M = sc.nextInt();
		 int L = sc.nextInt();
		 int happiness_score = S+2*M+3*L;
		 if (happiness_score>=10){
			 System.out.println("happy");
		 }else{
			 System.out.println("sad");
		 }
	}
}

File: 2020/C++/Junior Problems/J1.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    int S,M,L;
    cin >> S >> M >> L;
    int score = 1*S+2*M+3*L;
    if (score>=10){
        cout << "happy";
    }
    else{
        cout << "sad";
    }
    return 0;
}
